packages/config/config/devices/0x0312/templates/minoston_template.json

```diff
@@ -499,5 +499,67 @@
 		"maxValue": 20,
 		"defaultValue": 5,
 		"unsigned": true
-	}
+	},
+    "power_reporting": {
+		"label": "Power Reporting",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 1,
+		"defaultValue": 0,
+		"unsigned": true,
+        "allowManualEntry": false,
+		"options": [
+			{
+				"label": "Enabled",
+				"value": 0
+			},
+			{
+				"label": "Disabled",
+				"value": 1
+			}
+		]
+    },
+    "current_reporting": {
+        "label": "Current Reporting",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 1,
+		"defaultValue": 0,
+		"unsigned": true,
+        "allowManualEntry": false,
+		"options": [
+			{
+				"label": "Enabled",
+				"value": 0
+			},
+			{
+				"label": "Disabled",
+				"value": 1
+			}
+		]
+    },
+	"current_reporting_interval": {
+		"label": "Current Reporting Interval",
+		"valueSize": 4,
+		"unit": "minutes",
+		"minValue": 1,
+		"maxValue": 65535,
+		"defaultValue": 60,
+		"unsigned": true
+     },
+	"voltage_reporting_interval": {
+		"label": "Voltage Reporting Interval",
+		"valueSize": 4,
+		"unit": "minutes",
+		"minValue": 1,
```

Comment:
```suggestion
		"minValue": 0,
```
Otherwise the option "Disabled" is not a valid value.

Reply:
@AlCalzone I thought about that, and everything seemed to work the same either way.

I noted that when the configuration is loaded, and the parameter value is zero, it displays as zero. It doesn't dispkay as Disabled as I thought it should.  I'm assuming that it has to do with the allowManualEntry flag. The code below displays as Enabled or Diaabled. The code below that displays as zero.

My guess is that you know all about this, and that there isn't an easy fix.

```
	"current_reporting": {
		"label": "Current Reporting",
		"valueSize": 1,
		"minValue": 0,
		"maxValue": 1,
		"defaultValue": 0,
		"unsigned": true,
		"allowManualEntry": false,
		"options": [
			{
				"label": "Enabled",
				"value": 0
			},
			{
				"label": "Disabled",
				"value": 1
			}
		]
	}
```
```
	"voltage_reporting_interval": {
		"label": "Voltage Reporting Interval",
		"valueSize": 4,
		"unit": "minutes",
		"minValue": 1,
		"maxValue": 65535,
		"defaultValue": 60,
		"unsigned": true,
		"options": [
			{
				"label": "Disabled",
				"value": 0
			}
		]
	}
```

Reply:
That is supported already

options + Allow Manual Entry 🤷 


![Screenshot 2023-08-09 at 19 40 40](https://github.com/zwave-js/node-zwave-js/assets/55892693/0c964a8d-60ca-4434-abea-ff2f0b1bf890)

Reply:
@marcus-j-davies, I started looking into this but now for some reason my custom configuration file isn't getting loaded in. lm suspicious it has something to do with the upgrade you told me how to do yesterday.

Reply:
That was a driver update - not something that should stop custom configs from being loaded.

Driver logs will tell you if there are any problems.

Reply:
> now for some reason my custom configuration file isn't getting loaded in

Do you also have the template in your custom configuration dir?
Needs to have:
```
mp21zp_mp31zp.json
templates/minoston_template.json
```
before a recent update you also needed
```
templates/master_template.json
```
but that is now getting loaded from the internal db.

Reply:
I don't see anything wrong. I rebooted the Rpi, and reinterviewed the device.
```
2023-08-02T11:45:51.888Z NDERED   [options] [storage.cacheDir] /home/fabnav/.node-red/zwave-js-cache
2023-08-02T11:45:51.889Z NDERED   [options] [storage.deviceConfigPriorityDir] /home/fabnav/.node-red/zwave-custo
                                  m-config
```
```
fabnav@docker-hass:~/.node-red $ ls -R zwave-custom-config
zwave-custom-config:
mp21zp_mp31zp.json  templates

zwave-custom-config/templates:
minoston_template.json
```

From mp21zp_mp31zp.json
```
		{
			"#": "12",
			"$if": "firmwareVersion >= 1.11.1",
			"$import": "templates/minoston_template.json#current_reporting_interval"
		},
```

From minoston_template.json
```
	"current_reporting_interval": {
		"label": "Current Reporting Interval",
		"valueSize": 4,
		"unit": "minutes",
		"minValue": 1,
		"maxValue": 65535,
		"defaultValue": 60,
		"unsigned": true
	},
```
The label is supposed to be "Current Reporting Interval". It shows up as "Report frequency of currents".
![Untitled](https://github.com/zwave-js/node-zwave-js/assets/16625904/0fa0dc86-cf43-47b6-9b39-97265ccec28c)
r-hass:~/.node-red $

Reply:
A driver log will show if its being loaded, and if it encounters an error.
the `[options]` entries is what is passed to the driver, these are log entries I add - not the driver, further down will reveal activity by the driver its self.

Reply:
Uh, have you re-interviewed the device yet?

Reply:
What is a driver log? Do you just want the full zwave log?

Reply:
Driver log is a zwave log
zwave log is a Driver log

😄

Reply:
Why can't we just call it one thing? It's named "zwave-js.log". I vote for calling it zwave log.

Reply:
Here is the log of the interview.
[zwave.log](https://github.com/zwave-js/node-zwave-js/files/12314160/zwave.log)

Reply:
This is also interesting. I don't have any local files for nodes 108 and 109. I used to, but deleted the files before the previous reboot.
```
2023-08-02T11:45:56.455Z CNTRLR   [Node 001] Embedded device config loaded
2023-08-02T11:45:56.580Z CNTRLR   [Node 007] Embedded device config loaded
2023-08-02T11:45:56.591Z CNTRLR   [Node 016] Embedded device config loaded
2023-08-02T11:45:56.608Z CNTRLR   [Node 021] Embedded device config loaded
2023-08-02T11:45:56.618Z CNTRLR   [Node 022] Embedded device config loaded
2023-08-02T11:45:56.624Z CNTRLR   [Node 023] Embedded device config loaded
2023-08-02T11:45:56.631Z CNTRLR   [Node 024] Embedded device config loaded
2023-08-02T11:45:56.639Z CNTRLR   [Node 025] Embedded device config loaded
2023-08-02T11:45:56.645Z CNTRLR   [Node 027] Embedded device config loaded
2023-08-02T11:45:56.652Z CNTRLR   [Node 028] Embedded device config loaded
2023-08-02T11:45:56.658Z CNTRLR   [Node 029] Embedded device config loaded
2023-08-02T11:45:56.672Z CNTRLR   [Node 030] Embedded device config loaded
2023-08-02T11:45:56.682Z CNTRLR   [Node 044] Embedded device config loaded
2023-08-02T11:45:56.691Z CNTRLR   [Node 045] Embedded device config loaded
2023-08-02T11:45:56.696Z CNTRLR   [Node 048] Embedded device config loaded
2023-08-02T11:45:56.702Z CNTRLR   [Node 052] Embedded device config loaded
2023-08-02T11:45:56.710Z CNTRLR   [Node 055] Embedded device config loaded
2023-08-02T11:45:56.717Z CNTRLR   [Node 057] Embedded device config loaded
2023-08-02T11:45:56.724Z CNTRLR   [Node 058] Embedded device config loaded
2023-08-02T11:45:56.730Z CNTRLR   [Node 059] Embedded device config loaded
2023-08-02T11:45:56.739Z CNTRLR   [Node 100] Embedded device config loaded
2023-08-02T11:45:56.755Z CNTRLR   [Node 108] User-provided device config loaded
2023-08-02T11:45:56.780Z CNTRLR   [Node 109] User-provided device config loaded
2023-08-02T11:45:56.786Z CNTRLR   [Node 121] Embedded device config loaded
2023-08-02T11:45:56.792Z CNTRLR   [Node 122] Embedded device config loaded
2023-08-02T11:45:56.798Z CNTRLR   [Node 123] Embedded device config loaded
2023-08-02T11:45:56.805Z CNTRLR   [Node 124] Embedded device config loaded
2023-08-02T11:45:56.825Z CNTRLR   [Node 125] Embedded device config loaded
2023-08-02T11:45:56.830Z CNTRLR   [Node 133] Embedded device config loaded
2023-08-02T11:45:56.841Z CNTRLR   [Node 135] Embedded device config loaded
2023-08-02T11:45:56.864Z CNTRLR   [Node 142] Embedded device config loaded
2023-08-02T11:45:56.872Z CNTRLR   [Node 143] Embedded device config loaded
2023-08-02T11:45:56.879Z CNTRLR   [Node 144] Embedded device config loaded
2023-08-02T11:45:56.886Z CNTRLR   [Node 145] Embedded device config loaded
2023-08-02T11:45:56.894Z CNTRLR   [Node 146] Embedded device config loaded
2023-08-02T11:45:56.904Z CNTRLR   [Node 149] User-provided device config loaded
2023-08-02T11:45:56.911Z CNTRLR   [Node 150] Embedded device config loaded
2023-08-02T11:45:56.922Z CNTRLR   [Node 151] User-provided device config loaded
2023-08-02T11:45:56.940Z CNTRLR   [Node 152] Embedded device config loaded
2023-08-02T11:45:56.964Z CNTRLR   [Node 153] Embedded device config loaded
2023-08-02T11:45:56.974Z CNTRLR   [Node 154] User-provided device config loaded
2023-08-02T11:45:56.987Z CNTRLR   [Node 155] Embedded device config loaded
2023-08-02T11:45:56.994Z CNTRLR   [Node 156] Embedded device config loaded
2023-08-02T11:45:56.996Z CNTRLR   [Node 157] No device config found
2023-08-02T11:45:57.008Z CNTRLR   [Node 158] User-provided device config loaded
2023-08-02T11:45:57.016Z CNTRLR   [Node 159] Embedded device config loaded
2023-08-02T11:45:57.028Z CNTRLR   [Node 160] Embedded device config loaded
2023-08-02T11:45:57.044Z CNTRLR   [Node 161] Embedded device config loaded
```

Reply:
@AlCalzone 

```
2023-08-10T13:39:43.664Z CNTRLR   [Node 151] [Configuration] 12: metadata updated                   [Endpoint 0]
2023-08-10T13:39:43.664Z DRIVER « [Node 151] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -88 dBm
                                  └─[ConfigurationCCNameReport]
                                      parameter #:       12
                                      name:              Report frequency of currents
                                      reports to follow: 0
```

Do these reports take priority over user provided configs?
could be the reason if so 🤷 

I note : https://github.com/zwave-js/node-zwave-js/pull/6060 but that does also protect user provided?

Reply:
I feel this discussion should be moved if there is more to investigate.

Reply:
Wait for AlCalzone.